tests/e2e/scripts/cloud-run/deploy.sh

```diff
@@ -63,6 +64,14 @@ if [[ "${PROXY_PLATFORM}" == "anthos-cloud-run" ]] ; then
   CLUSTER_NAME=$(get_anthos_cluster_name_with_sha)-${UNIQUE_ID}
 fi
 
+GCLOUD_BETA="gcloud"
+USE_HTTP2=""
+# For grpc echo test, need to use gcloud beta with --use-http2 flag
```

Comment:
NIT.

# For grpc echo test, need to use gcloud beta with --use-http2 flag because it covers grpc streaming cases.

Reply:
Done